ITE/4/README.adoc

```diff
@@ -0,0 +1,462 @@
+expects in every step of the supply chain in the in-toto layout. in-toto
+provides the following rules that together attempt to describe the different
+types of operations or actions that supply chain functionaries carry out.
+
+*MATCH Rule*
+
+`MATCH <pattern> [IN <source-path-prefix>] WITH (MATERIALS|PRODUCTS) [IN
+<destination-path-prefix>] FROM <step>`
+
+The `MATCH` rule is a convenient way to match artifacts (either in materials or
+products depending on where the rule is specified) with artifacts from other
+steps in the supply chain, allowing owners to establish a flow of artifacts
+between steps in the software supply chain.
+
+The current verification workflow for the `MATCH` rule compares the
+cryptographic hashes of the relevant artifacts from the respective link files.
+This is a straightforward comparison and does not entail resolving the URIs in
+any form. The `IN` clauses which are used to specify path prefixes are
+subtracted from the URI strings. Therefore, we conclude that the functioning of
+the rule doesn't change with this ITE. An example `MATCH` rule with generic URIs
+is as follows:
+
+`MATCH commit/* IN github+https://github.com/in-toto/in-toto/ WITH PRODUCTS IN
+github+https://github.com/in-toto/in-toto FROM merge-pull-request-250`
+
+*Other artifact rules*
+
+in-toto provides several other artifact rules - specifically `ALLOW`,
+`DISALLOW`, `CREATE`, `DELETE`, `REQUIRE`, and `MODIFY`. These rules perform
+different checks by verifying if artifacts matching the pattern occur or do not
+occur in the materials or products sections of the relevant link. This does not
+involve resolving the pattern or URI into the artifact itself, and so this ITE
+does not affect the working of these rules.
+
+[[motivation]]
+== Motivation
+
+ITE-4 is motivated by the following use cases.
+
+==== Use Case 1: SPDX Document Identifiers
+
+Software Package Data Exchange (SPDX) is an open standard for communicating
+software bill of materials (SBoM) information such as components, licenses,
+copyrights, and security references. Each document is a comprehensive report
+that describes a software package in detail. SPDX is a part of the broader
+discussions with Continuous Delivery (CD) Foundation's
+link:https://github.com/cdfoundation/sig-security-sbom[Special Interest Group for Software Bill of Materials],
+as well as with the National Telecommunication and Information Administration's
+(NTIA) link:https://www.ntia.doc.gov/SoftwareTransparency[Software Component Transparency].
+SPDX and in-toto, along with representatives from NTIA and other stakeholders,
+are also part of the Consortium for Information and Software Quality (CISQ) and
+Object Management Group's (OMG) working group on
+link:https://www.it-cisq.org/software-bill-of-materials/index.htm[Tool-to-tool Software Bill of Materials Exchange].
+
+SPDX documents composed of several entities that have unique identifiers. These
```

Comment:
> SPDX documents composed of several entities that have unique identifiers.

missing "are"  (I can't suggest the edit, possibly because I'm typing this while on a flight.)